src/dynamic_boundary_conditions/hirds_depth_data_from_db.py

```diff
@@ -6,75 +6,75 @@
 """
 
 import pandas as pd
-import geopandas
+import pathlib
+import logging
+import sys
+from shapely.geometry import Polygon
 from src.dynamic_boundary_conditions import hirds_depth_data_to_db
+from src.digitaltwin import setup_environment
+from src.dynamic_boundary_conditions import hyetograph
 
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
 
-def catchment_area_geometry_info(file):
-    """Extract geometry from the shape file, returns polygon."""
-    catchment = geopandas.read_file(file)
-    catchment = catchment.to_crs(4326)
-    catchment_area = catchment.geometry[0]
-    return catchment_area
-
-
-def get_each_site_hirds_depth_data(
-    ari, duration, site, engine, rcp=None, time_period=None
-):
-    """Get hirds rainfall depth data from the database."""
-    if (rcp is None and time_period is not None) or (
-        rcp is not None and time_period is None
-    ):
-        raise ValueError(
-            "check the arguments of get_hirds_depth_data if rcp is None,time"
-            " period should be None and vice-versa"
-        )
+formatter = logging.Formatter("%(levelname)s:%(asctime)s:%(name)s:%(message)s")
+stream_handler = logging.StreamHandler()
+stream_handler.setFormatter(formatter)
+
+log.addHandler(stream_handler)
```

Comment:
Good to see that you are using logging. There may be ways to set the level in your run configurations, if you find out how then you may be able to remove  `logging.setLevel` call here